client/tso_service_discovery.go

```diff
@@ -160,7 +297,7 @@ func (c *tsoServiceDiscovery) startCheckMemberLoop() {
 			log.Info("[tso] exit check member loop")
 			return
 		}
-		if err := c.updateMember(); err != nil {
+		if err := c.retry(tsoQueryRetryMaxTimes, tsoQueryRetryInterval, c.updateMember); err != nil {
```

Comment:
Previously, we only update members once, but now we retry many times. Will retry blocks checkMembershipCh?

Reply:
it won't. the memberUpdateInterval of loop is 1m, while tsoQueryRetryMaxTimes is 10 and tsoQueryRetryInterval is 500ms (5s in total in retries) to speed up member discovery. I'll add a comment saying "tsoQueryRetryMaxTimes * tsoQueryRetryInterval should be far less than memberUpdateInterval"